Fix Failing CI/CD for Windows

See https://github.com/vsaw/grid-at-a-glance/actions/runs/13884517132/job/38847410728

```
✖ Making a squirrel distributable for win32/x64 [FAILED: You must install both Mono and Wine on non-Windows]
✖ Making distributables [FAILED: You must install both Mono and Wine on non-Windows]

An unhandled rejection has occurred inside Forge:
Error: You must install both Mono and Wine on non-Windows
at /home/runner/work/grid-at-a-glance/grid-at-a-glance/node_modules/electron-winstaller/lib/index.js:128:31
    at step (/home/runner/work/grid-at-a-glance/grid-at-a-glance/node_modules/electron-winstaller/lib/index.js:56:23)
    at Object.next (/home/runner/work/grid-at-a-glance/grid-at-a-glance/node_modules/electron-winstaller/lib/index.js:37:53)
    at fulfilled (/home/runner/work/grid-at-a-glance/grid-at-a-glance/node_modules/electron-winstaller/lib/index.js:28:58)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
25h
Error: Process completed with exit code 1.
```

--- a/.github/workflows/ci-cd.yml
+++ b/.github/workflows/ci-cd.yml
@@ -31,9 +31,12 @@ jobs:
       - uses: actions/checkout@v4
       - name: Install Wine and Mono for Windows
         if: ${{ matrix.platform == 'win32' }}
+        # FIXME: Work around missing wine64 binary as suggested in
+        # https://github.com/electron/forge/issues/3778
         run: sudo dpkg --add-architecture i386 &&
           sudo apt-get update &&
-          sudo apt-get install wine mono-devel
+          sudo apt-get install wine wine64 mono-complete &&
+          sudo ln -s /usr/bin/wine /usr/bin/wine64
       - name: Use Node.js ${{ matrix.node-version }} - ${{ matrix.architecture }}
         uses: actions/setup-node@v4
         with:

